iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/AffMatchingJob.java

```diff
@@ -112,12 +118,31 @@ private static AffMatchingService createAffMatchingService(JavaSparkContext spar
         AffOrgMatcher firstWordsHashBucketMatcher = createFirstWordsHashBucketMatcher();
         
         
+        affMatchingService.setAffOrgMatchers(of(docOrgRelationMatcher, mainSectionHashBucketMatcher, firstWordsHashBucketMatcher));
         
         
-        affMatchingService.setAffOrgMatchers(of(docOrgRelationMatcher, mainSectionHashBucketMatcher, firstWordsHashBucketMatcher));
+        AffMatchOrganizationAltNameFiller altNameFiller = createAffMatchOrganizationAltNameFiller();
+        affMatchingService.setAffMatchOrganizationAltNameFiller(altNameFiller);
         
         return affMatchingService;
     }
     
+    private static AffMatchOrganizationAltNameFiller createAffMatchOrganizationAltNameFiller() throws IOException {
+        
+        AffMatchOrganizationAltNameFiller altNameFiller = new AffMatchOrganizationAltNameFiller();
+        
+        List<String> altNamesCountryCodes = of(
```

Comment:
IMO it is overcomplication.
If someone is adding new file with alternative names pack in `src/main/resources/eu/dnetlib/iis/wf/affmatching/universities` then it is implied that he knows about definition in java code because he had to somehow find this files. Assuming standard approach for exploration of the code (that is top-down approach) developer will go through `workflow.xml` ->`AffMatchingJob` -> `createAffMatchOrganizationAltNameFiller()` and he would find location of the all alternative names files. Next he would think: 'All the files are under `src/main/resources/eu/dnetlib/iis/wf/affmatching/universities` classpath. Ok, I will add new file there`.
As for definition existing in several places - I will fix it.